pkg/dynamicinstrumentation/ditypes/analysis.go

```diff
@@ -400,7 +417,7 @@ type BPFProgram struct {
 //nolint:all
 func PrintLocationExpressions(expressions []LocationExpression) {
 	for i := range expressions {
-		fmt.Printf("%s %d %d %d %s %s\n",
+		fmt.Printf("Opcode: %s Args: [%d, %d, %d] Label: %s Collection ID: %s\n",
```

Comment:
Is this for debugging? You could attach this as a `String() string` to `LocationExpression`

Reply:
The funny thing is that I already had a String() method for LocationExpression, I just forgot about it and wasn't using it :-)

Reply:
And yes, purely for debugging. The nolint comment is because I use it during development. Hard to get the same information when stepping through in delve.